src/main/java/core/basesyntax/model/InvalidDataException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.model;
+
+public class InvalidDataException extends Exception {
```

Comment:
You should create your custom **unchecked** exception, but your exception is checked